docs/user/alerting/rule-types/geo-rule-types.asciidoc

```diff
@@ -2,11 +2,9 @@
 [[geo-alerting]]
 === Geo rule type
 
-Alerting now includes one additional stack rule: <<rule-type-tracking-containment>>.
-
-As with other stack rules, you need `all` access to the *Stack Rules* feature
-to be able to create and edit a geo rule.
-See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to this feature. 
+The Tracking containment rule type runs an {es} query over indices, determining if any
```

Comment:
This is much better, but we still call the rule by three different names:  tracking containment, geo rule, and map rule (this one is on the previous page). Not sure what the correct name is.

Also, there are a few instances on the page where "tracking containment" is in bold.  Since we don't put the name in bold on the other pages, I think we should remove it here.

Reply:
I renamed the page `Tracking containment` and updated the intro to make clear it was a `Maps` rule. Previously, Maps offered two "Geo type" rules and when they removed one, they just removed that section, which is why I think some of the wording was confusing. 

Also removed the unnecessary bolding.